user-journey.adoc

```diff
@@ -3,20 +3,26 @@
  *Authors:* Lincoln Baxter, III and Charles Moulliard, Diógenes Rettori
  *Version:* 0.6 - April 25, 2018
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites (JDK, Maven, Launcher, Jaeger, …), and the operations to grant access for the user to the project. It also describes how the user can experiment with a mission selected using the launcher
-
-The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
 
 == Flows description
 
-*“Zip flow”:* Selected mission is downloaded from the web site as a zip archive and extracted to the local filesystem.
-*“Git flow”:* Selected mission is saved to a github repository/account, and an Istio application is created under the user’s local Openshift instance - within the project defined (which corresponds to the same name as the git repo created).
+The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+
+. *`Zip flow`:* Selected mission is downloaded from the web site as a zip archive and extracted to the local filesystem.
+. *`Git flow`:* Selected mission is saved to a GitHub repository/account, and an Istio enabled application is created in a local Openshift instance.
 
 == Prerequisites
-. *Access to a docker engine - (skip to step 2 if you already have one)*
-.. Red Hat Docker Registry - Certificate Configuration
+. *Install Java JDK 8* (necessary only for `Zip flow` and boosters using Fabric8 Maven Plugin)
+.. http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[JDK 8}
```

Comment:
Typo, the closing bracket is a curly brace, so the link doesn't format correctly.

Reply:
Thanks. Fixed.